nservicebus/upgrades/externalize-encryption.md

```diff
@@ -36,4 +35,23 @@ snippet: SplitDefiningEncryptedPropertiesAs
 
 ## Custom encryption Service
 
-snippet: SplitEncryptionFromIEncryptionService
\ No newline at end of file
+snippet: SplitEncryptionFromIEncryptionService
+
+
+## Compatibility
+
+The NServiceBus.Encryption.MessageProperty package is partially compatible with endpoints using NServiceBus package's encryption functionality:
+
+
+### Sending and receiving using NServiceBus.Encryption.MessageProperty
```

Comment:
> Sending and receiving using NServiceBus.Encryption.MessageProperty

Do we actually send and receive messages with this package, or should it be "Sending and receiving messages with properties encrypted using `NServiceBus.Encryption.MessageProperty`" or something like that?